### ISSUE TITLE
Logs Crash when Trying to Grab Them

### ISSUE THREAD

**Issue by MetalNeo** [FreeTubeApp/FreeTube#786]
**Behavior of the program**
When trying to view/copy the logs for another issue I am having, the logging area becomes sluggish when interacting with it after the issue occurs and will then turn completely blank/white. 
![White Logging Area](https://user-images.githubusercontent.com/53447663/99151915-4c2e8880-2696-11eb-88b7-2b5359bea022.png)

**Expected behavior**
The logs stay responsive and don't turn blank.

**To Reproduce**
Please add all steps to reproduce the behavior:
1. Open up logs with Ctrl + Shift + I.
2. Click a video to start playing it.
3. An excessive amount of the same 2 errors appear in the logs before becoming sluggish, and then turning white.

**Screenshots**
These two errors appear over and over again before logs become sluggish/turn white.
[-1605371685817.log](https://github.com/FreeTubeApp/FreeTube/files/5541105/-1605371685817.log)
[-1605372111019.log](https://github.com/FreeTubeApp/FreeTube/files/5541107/-1605372111019.log)

**Environment Information (please complete the following information):**
 - OS: Arch Linux
 - OS Version: Rolling
 - FreeTube version: Git (commit 1357)
 - Installation Method (If applicable): AUR package

**Additional context**
This issue occurred when trying to grab logs for #778.

**Comment by GilgusMaximus** [comment]
To me this looks like a Electron problem. Maybe something that can be changed in the setup of Electron, however I did not mess with the setting. But I encountered the problem of logs taking forever to save into a file in other Electron apps as well.

**Comment by MetalNeo** [comment]
It doesn't seem to be an issue with the logging feature itself. Before I click a video, the log responds normally and is responsive to my interactions. It is only after clicking a video that the issue seems to manifest.

**Comment by MetalNeo** [comment]
I have discovered a workaround to this issue. Ctrl + shift + i can be pressed after the logs go white to remove the log panel, then pressed again to bring the logs back up visibly. So far, this has allowed me to collect error logs every time an error has occurred.

**Comment by MetalNeo** [comment]
I have fixed the original issue that was causing the logger to crash and found a way to reload the logger in case it crashes again, so I believe this issue should be good to close.
